sdk/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -419,7 +420,9 @@ public static void registerSession(long startedAt, @Nullable String sessionId,
      *                     stages
      */
     @SuppressWarnings("unused")
-    public static void deliverReport(@Nullable String releaseStage, @NonNull String payload) {
+    public static void deliverReport(@Nullable byte[] releaseStageBytes, @NonNull byte[] payloadBytes) {
```

Comment:
There are several other methods in `NativeInterface` which take a `String` as a parameter. Are they also affected by this issue?

Reply:
Yeah, that's a good call. There are only a few which are called from the NDK interface, and those are:

* `deliverReport`
* `leaveBreadcrumb`
* `setUser`
* `notify`

I've added additional APIs to cover the others so they can be called with byte arrays. Going forward, we should have a linting rule to disallow calling string methods from the native interface without an annotation (something I can look into adding later).

Reply:
Sounds like a good approach.